packages/core/src/js/options.ts

```diff
@@ -294,7 +294,7 @@ export interface ReactNativeClientOptions
 
 export interface ReactNativeWrapperOptions {
   /** Props for the root React profiler */
-  profilerProps?: ProfilerProps;
+  profilerProps?: Pick<Partial<ProfilerProps>, Exclude<keyof ProfilerProps, 'updateProps' | 'children' | 'name'>>;
```

Comment:
Both variants have the same result with the current `ProfilerProps`, but with Omit is a bit shorter.

````suggestion
profilerProps?: Omit<ProfilerProps, 'updateProps' | 'children' | 'name'>
````